### PR TITLE
Fallback to nodejs fs methods on non-linux platforms

### PR DESCRIPTION
**Pull request by CheerlessCloud**

### CODE REVIEW

File: lib/utimesSync.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+module.exports = function utimesSync(pathToFile, atime_nsec, mtime_nsec) {
+  atime_nsec = atime_nsec.toString().replace('.', '');
+  mtime_nsec = mtime_nsec.toString().replace('.', '');
+  try {
+    return fstime.utimesSync(
+      pathToFile,
+      parseInt(atime_nsec.substr(0, 10)), // atime_sec
+      parseInt(atime_nsec.substr(10, 9)) || 0, // atime_nsec 880493406
+      parseInt(mtime_nsec.substr(0, 10)), // mtime_sec
+      parseInt(mtime_nsec.substr(10, 9)) || 0, // mtime_nsec 880493406
```
Comment (Goodluckhf) [inline lib/utimesSync.js]: I think you have to specify numeral system `10`

File: lib/utimesSync.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    );
+  } catch (err) {
+    err.message = err.message + `: ${pathToFile}`;
```
Comment (Goodluckhf) [inline lib/utimesSync.js]: here you can put `'${err.message}: ${pathToFile}';`

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,3120 @@
+{
```
Comment (Goodluckhf) [inline package-lock.json:1]: Are you sure you need this file?

File: lib/utimesSync.js (hunk 1 above)
Comment (CheerlessCloud) [reply lib/utimesSync.js]: yep, I'll fix it soon

File: lib/utimesSync.js (hunk 2 above)
Comment (CheerlessCloud) [reply lib/utimesSync.js]: Hmmm... It was an automatic fix 😁

**Pull request merged**